HEATDamage.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using System.Text;
+using System.Diagnostics;
+
+namespace PhysicalDamage.Core
+{
+    class HEATDamage:Damage
+    {
+        public struct HEATData
+        {
+            public float Pierce;
+
+            public float ArmorHEATFactor;
+
+            public float HealthHEATFactor;
+        }
+
+        private HEATData data;
+
+        public HEATDamage(HEATData d,Damage.Target t):base(EDamageTypes.HEAT, t)
```

Comment:
Rename constructor to `HeatDamage` and give more meaningful variable names as well as spacing.

`public HeatDamage(HeatData heatData, Damage.Target target) : base(DamageTypes.HEAT, target)`